test/config/mocks/reset-dynamic-directive-injections.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Angular providers do not reset between each test, only service instance do.
+ */
+(function(angular) {
```

Comment:
Please create a commit to remove IIFE whenever possible so that in the future we don't have to do it again :D

Reply:
Fixed